lib/manageiq/session.rb

```diff
@@ -59,7 +59,10 @@ def self.configure_session_store(adapter)
     # of session ids provided and sent to be delete from the session @store.
     #
     def self.revoke(*session_ids)
-      store.delete_sessions(session_ids.flatten)
```

Comment:
This seems kind of unrelated - can it be a separate PR because it feels like a no-brainer merge